lsusb.c

```diff
@@ -288,6 +288,32 @@ static void dump_device(
 		read_sysfs_prop(serial, sizeof(serial), sysfs_name, "serial");
 	}
 
+    int speed = libusb_get_device_speed(dev);
+    if (speed != LIBUSB_SPEED_UNKNOWN) {
```

Comment:
Coding style is tabs (that are 8 characters wide, like the Linux kernel coding style rules), this seems to be spaces, can you fix that up please?

Reply:
Done